project/SparkBuild.scala

```diff
@@ -1085,7 +1085,7 @@ object ExcludedDependencies {
     excludeDependencies ++= Seq(
       ExclusionRule(organization = "com.sun.jersey"),
       ExclusionRule(organization = "ch.qos.logback"),
-      ExclusionRule("javax.ws.rs", "jsr311-api"))
+      ExclusionRule("jsr311-api"))
```

Comment:
I read the above comment and the original PR, looks like this part of dependency management can be simplified independently, let me try.

Reply:
I opened https://github.com/apache/spark/pull/48468 to simplify the `yarn` module dependency management